src/app/common/services/auth-service.service.ts

```diff
@@ -12,16 +12,23 @@ import { MOCK_USER } from '../mock/user';
 })
```

Comment:
the filename is strange `auth-service.service.ts` it should be `auth.service.ts`